manage.sh

```diff
@@ -105,6 +105,8 @@ DEFAULT_SERVICES=(
 	stacks-blockchain
 	stacks-blockchain-api
 	postgres
+# Uncomment the following line to export also bitcoin-core logs for troubleshooting
```

Comment:
not relevant, should be removed - bitcoin is out of scope of the main services this repo *requires* to run, only stacks-blockchain, stacks-blockchain-api, postgres are required services.

Reply:
Ok, removed comment.